### DISCUSSION TITLE
Binding Failed

### DISCUSSION THREAD

**Discussion by EhmigFlorian** [COVESA/dlt-viewer#572] [Q&A]
Hello so the basic setup which I have right now is, that I connected an external device to my PC via an ethernet to USB-adapter. The extern device works fine and can send out DLT frames on the ethernet port which I mentioned. I already checked via Wireshark. The problem now is that I am no able to connect the DLT Viewer to the Ethernet port via UDP.  When I try to connect, it always says “Binding Failed”. I checked that the IP-Port is set up correct and that no application uses the Ethernet-Port. I also tried to debug the DLT Viewer via the QT Creator and unfortunately the only result I got was “Unknown Error”. The only other problem I though off was that I got some kind of problem with my Firewall. So, my question would be, what that are normally means and what I can do to get it running. Maybe someone already had a similar problem. I am currently using Version 2.27.0 on Win11. Thanks in advance.

![image](https://github.com/user-attachments/assets/fc10c208-964b-4a90-8664-4fcf841cc31a)

**Comment by vifactor** [comment]
I was not able to connect to dlt-daemon using UDP neither, see https://github.com/COVESA/dlt-viewer/issues/523

**Comment by NicklasAfry** [comment]
Hello! I've got the exact same issue.

I'm running DLT Viewer on Windows 11 and I've tried version 2.28.1 (stable) and 2.29.0 (unstable) on one laptop and I always get "binding failed". I've also checked with netstat that the port (2896) is not already in use.

I've also tried version 2.30.0 on another laptop (also on Windows 11, but in case there are configuration/firewall issues on my corporate laptop) and I get the exact same result.

I've also checked with Wireshark on both laptops and I can confirm using Wireshark that the UDP packets are there.

Have you been able to resolve the issue somehow?
